### PR TITLE
BaSP-TG: Create and edit employee

### PR DESCRIPTION
**Pull request by GuidoG95**

### CODE REVIEW

File: src/resources/employees.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const newEmployeeKeys = Object.keys(newEmployee);
+  if (newEmployeeKeys.length < 6) {
+    res.send('Please complete all fields');
+  } else {
+    employees.push(newEmployee);
+    fs.writeFile('src/data/employees.json', JSON.stringify(employees), (err) => {
+      if (err) {
+        res.send('New user not saved.');
+      } else {
+        res.send('New user created.');
```
Comment (ignacioaristo) [inline src/resources/employees.js:18]: We could show the data of the new user too. For e.g: 
`res.send('New user created: ${newEmployee}');`

File: src/resources/employees.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const fs = require('fs');
+const employees = require('../data/employees.json');
+
+const router = express.Router();
+
+router.post('/add', (req, res) => {
```
Comment (ignacioaristo) [inline src/resources/employees.js:7]: I could pass any key, for e.g:

>{
    "id": 1,
    "asd": "asd",
    "asddas": "asd",
    "asdasdasd": "asd",
    "asdd2323": "asd",
    "sdsdsd": "asd"
}

And this employee will be created. We need to handle better this validation.

**Review by ignacioaristo — DISMISSED** [review]

File: src/index.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ const port = process.env.PORT || 3000;
 
 app.use(express.json());
 
+app.use('/employees', require('./resources/employees'));
```
[the remark this answers is not in the document]
Comment (GuidoG95) [reply src/index.js:12]: you are suggesting to use it like:
const employees = './resources/employees';
app.use('employees' , require(employees));
if not, i think i don't understand what you are trying to say

[the remark this answers is not in the document]
Comment (ignacioaristo) [reply src/index.js:12]: Almost, 

> const employees = './resources/employees';
app.use('/employees' , employees);

File: src/index.js [hunk 4]
```diff
@@ -13,6 +15,8 @@ app.use('/admins', adminRouter);
 app.use('/timeSheet', timeSheetRouter);
 app.use('/projects', projects);
```
Comment (baudraccoverona) [inline src/index.js:17]: Delete this empty line

**Review by baudraccoverona — CHANGES_REQUESTED** [review]

**Review by baudraccoverona — APPROVED** [review]

**Pull request merged**